kubernetes/images/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:latest
+MAINTAINER Ansgar Schmidt <ansgar.schmidt@gmx.net>
+ENV DEBIAN_FRONTEND noninteractive
+
+ARG BRANCH
+
+# update
+RUN apt-get update
+RUN apt-get upgrade -y
+
+# add packages
+RUN apt-get install -y git openjdk-8-jdk
+
+# clone the github repo
+RUN git clone https://github.com/fossasia/susi_server.git /susi_server
+WORKDIR /susi_server
+RUN git submodule update --recursive --remote
+RUN git submodule update --init --recursive
+RUN git checkout $BRANCH
```

Comment:
You're only checking out the branch. Where are you taking care of the current commit hash to ensure the code in the image is the same as the commit hash being tagged with the image ?
Something like
https://github.com/fossasia/open-event-orga-server/blob/nextgen/kubernetes/images/web/setup.sh#L7